Description: Fix segfault in is_cond_binary_op when compiling with -fPIE
 The issue possibly also pops up under other circumstances.
Author: Bart Schaefer <schaefer@brasslantern.com>
Origin: http://www.zsh.org/mla/workers/2016/msg00394.html

Index: zsh/Src/text.c
===================================================================
--- zsh.orig/Src/text.c	2016-01-24 17:19:53.947656591 +0100
+++ zsh/Src/text.c	2016-02-01 00:15:09.098327625 +0100
@@ -47,7 +47,7 @@
  */
 static const char *cond_binary_ops[] = {
     "=", "!=", "<", ">", "-nt", "-ot", "-ef", "-eq",
-    "-ne", "-lt", "-gt", "-le", "-ge", "=~"
+    "-ne", "-lt", "-gt", "-le", "-ge", "=~", NULL
 };
 
 static char *tptr, *tbuf, *tlim, *tpending;
